pkg/reporter/markdown.go

```diff
@@ -45,6 +47,12 @@ type markdownTemplateInput struct {
 	Violations     []markdownTemplateInputViolation
 	ManifestsCount int
 	PackagesCount  int
+	CriticalVulnCount int
```

Comment:
Non-blocking: Looks like we are combining two features in a single PR i.e. CycloneDX SBOM support and Markdown reporting enhancement. I think let us have individual PR for each feature in future so that we can track which PR introduced what.

Reply:
I will do it from next time